pysb/examples/schlogl.py

```diff
@@ -0,0 +1,20 @@
+from pysb import *
```

Comment:
Please add a description and reference for the model

Reply:
Done